rhel6/checks/oval/file_group_owner_grub_conf.xml

```diff
@@ -1,37 +0,0 @@
-<def-group>
-  <definition class="compliance" id="file_group_owner_grub_conf" version="1">
-    <metadata>
-      <title>File grub.conf Owned By root Group </title>
-      <affected family="unix">
-        <platform>Red Hat Enterprise Linux 6</platform>
-      </affected>
-      <description>The grub.conf file should be owned by the root group. By default, this file is located at /boot/grub/grub.conf or, for EFI systems, at /boot/efi/EFI/redhat/grub.conf</description>
```

Comment:
I can't grep any mention of EFI in `ssg-rhel6-oval.xml`. Why was the EFI alternative removed?